### PR TITLE
Correctly style tooltips in devhub

### LINKED ISSUE #2467: Tooltip text not correctly aligned after pressing Edit in Edit listing section
Prerequisites:
Build identifier: Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:45.0) Gecko/20100101 Firefox/45.0

Steps to reproduce:
1.Go to edit listing page for an addon (https://addons.allizom.org/en-US/developers/addon/6april/edit)
2.Observe the tooltips before pressing Edit button for a section
3.Press Edit button for a section(eg. Techincal Details)
4.Observe the tooltips again

Expected results:
Tooltips are corectly aligned

Actual results:
Tooltips not aligned

NOTE:Not reproducible on prod

Please see video for this issue : 
tooltip

### PR DESCRIPTION
**Pull request by wagnerand**
Fixes #2467

### CODE REVIEW

**Comment by tofumatt** [comment]
Could we have screenshots so I/QA can see the diff and confirm it?

r+wc

File: static/js/zamboni/devhub.js [hunk 1]
```diff
@@ -283,6 +283,7 @@ $(document).ready(function() {
     $(document).on('click', '.addon-edit-cancel', function(){
         parent_div = $(this).closest('.edit-addon-section');
         parent_div.load($(this).attr('href'), function() {
+            $(".tooltip").tooltip("#tooltip");
```
Comment (tofumatt) [inline static/js/zamboni/devhub.js]: Nitpick but please use single quotes; they're used everywhere else.

Comment (wagnerand) [inline static/js/zamboni/devhub.js]: Meh, I just copy-pasted https://github.com/mozilla/addons-server/blob/master/static/js/zamboni/global.js#L87

**Comment by wagnerand** [comment]
postfix

**Pull request merged**